libraries/lib-audio-devices/AudioIOBase.cpp

```diff
@@ -802,7 +802,7 @@ int AudioIOBase::getRecordDevIndex(const wxString &devNameArg)
       // JKC: This ASSERT will happen if you run with no config file
```

Comment:
Could you also update comments to reflect changes?

Reply:
fixed